packages/browser-utils/src/instrument/xhr.ts

```diff
@@ -75,6 +85,7 @@ export function instrumentXHR(): void {
             endTimestamp: timestampInSeconds() * 1000,
             startTimestamp,
             xhr: xhrOpenThisArg,
+            error: httpClientInstrumented ? virtualError : undefined,
```

Comment:
m: Can we just directly attach the stackframe here, instead of passing the error around? then we can maybe just ignore these checks and simply always attach this here 🤔

Reply:
Yes, this made it work with ESM builds. Thanks!